File: sentimental-cash/cash.py
from cs50 import get_float

quarter = 25
dime = 10
nickel = 5
penny = 1

coins = 0

while True:
    amount = get_float("Change: ")
    if amount > 0:
        break

amount = round(amount * 100)

coins += amount // quarter
amount %= quarter

coins += amount // dime
amount %= dime

coins += amount // nickel
amount %= nickel

coins += amount // penny

print(int(coins))

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    const int QUARTER = 25;
    const int DIME = 10;
    const int NICKEL = 5;
    const int PENNY = 1;
    int amount;
    int coins = 0;

    while (true)
    {
        amount = get_int("Change owed: ");

        if (amount > 0)
        {
            break;
        }
    }

    coins += amount / QUARTER;
    amount %= QUARTER;

    coins += amount / DIME;
    amount %= DIME;

    coins += amount / NICKEL;
    amount %= NICKEL;

    coins += amount;

    printf("%d\n", coins);

    return 0;
}
